### PR TITLE
Move OccupancySensing to match the spec

### PR DESCRIPTION
**Pull request by andy31415**
Changes:
  - marked 2 properties as nullable to match the spec

### CODE REVIEW

File: src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml [hunk 1]
```diff
@@ -89,13 +89,13 @@ limitations under the License.
       <access op="write" role="manage"/>
     </attribute>
 
-    <attribute side="server" code="0x0030" define="PHYSICAL_CONTACT_OCCUPIED_TO_UNOCCUPIED_DELAY" type="int16u" writable="true" default="0x0000" optional="true">
+    <attribute side="server" code="0x0030" define="PHYSICAL_CONTACT_OCCUPIED_TO_UNOCCUPIED_DELAY" type="int16u" writable="true" default="0x0000" optional="true" isNullable="true">
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml:92]: This is a backwards incompatible API and wire change.  We need to change the spec instead.

**Review by bzbarsky-apple — CHANGES_REQUESTED** [review]

File: src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml (hunk 1 above)
Comment (ReneJosefsen) [reply src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml:92]: Looking at the spec, this is also a bit strange. The PhysicalContactOccupiedToUnoccupiedDelay and PhysicalContactUnoccupiedToOccupiedDelay are the only delay attributes which is nullable.

I can make a PR to remove the X quality and remove "The null value indicates that the sensor does not report occupied to unoccupied transition." statement from the attributes? @andreilitvin?

Comment (andy31415) [reply src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml:92]: I think we should change the spec here: make things consistent and also not break backwards wire compatibility (which we certainly cannot do)

Comment (ReneJosefsen) [reply src/app/zap-templates/zcl/data-model/chip/occupancy-sensing-cluster.xml:92]: https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7799

**Comment by andy31415** [comment]
Closing, keeping spec change instead only.

**Pull request closed without merging**